Sources/Plasma/FeatureLib/pfJournalBook/pfJournalBook.h

```diff
@@ -513,8 +533,9 @@ class pfJournalBook : public hsKeyedObject
         // Some animation keys we use
         plKey   fPageTurnAnimKey;
 
-        // Current list of linkable image chunks we have visible on the screen, for quick hit testing
-        std::vector<pfEsHTMLChunk *> fVisibleLinks;
+        // Current list of visible link hotspots for quick hit testing.
+        // Can be images or lines of text.
+        std::vector<class pfJournalVisibleLink> fVisibleLinks;
```

Comment:
Inline type declarations are eeeehhhhh if you ask me... could we declare this together with all the other classes, at the top of the file?